### PR TITLE
[#OCD-3234] Add Self-Developer element to Developer information

### PR DESCRIPTION
**Pull request by andlar**

### CODE REVIEW

File: chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/DeveloperController.java [hunk 1]
```diff
@@ -293,6 +293,7 @@ public PermissionDeletedResponse deleteUserFromDeveloper(
             toCreate.setDeveloperCode(developerInfo.getDeveloper().getDeveloperCode());
             toCreate.setName(developerInfo.getDeveloper().getName());
             toCreate.setWebsite(developerInfo.getDeveloper().getWebsite());
+            toCreate.setSelfDeveloper(developerInfo.getDeveloper().getSelfDeveloper());
```
Comment (kekey1) [inline chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/DeveloperController.java]: I think there should be an additional change in this file... in the split method?

**Comment by andlar** [comment]
I don't think the Self-developer field should go into the upload/confirm workflow yet. It's not in the upload template, which means it won't be in the pending_certified_product_details, and while I could add it to the developer associated with the pending listing, there's no way to edit that developer to set it to be anything other than "No" until after the Listing has been confirmed, and the new Developer has been created. When it has, it'll be created as "No", and can then be edited

File: chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/DeveloperManager.java [hunk 2]
```diff
@@ -389,11 +389,21 @@ public DeveloperDTO merge(List<Long> developerIdsToMerge, DeveloperDTO developer
     }
 
     /**
+<<<<<<< HEAD
```
Comment (kekey1) [inline chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/DeveloperManager.java]: Conflict?

Comment (andlar) [reply chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/DeveloperManager.java]: Ugh. Eclipse didn't show it to me, and why would I look for conflicts in documentation that I didn't touch? Sigh

**Pull request merged**